en/gps_compass/rtk_gps.md

```diff
@@ -140,6 +157,70 @@ See the [EKF2 GPS Configuration](../advanced_config/tuning_the_ecl_ekf.md#gps) s
 -->
 
 
-## Vehicle Setup Example
+### Vehicle Setup Example
 
 The airframe build topic [DJI Flamewheel 450 with distance sensor and RTK GPS](../frames_multicopter/dji_flamewheel_450.md) describes an airframe setup with the Here+ RTK GPS and a Pixhawk 3 Pro.
+
+
+## RTK GPS Heading Setup/Configuration (U-blox F9P)
+
+Two u-blox F9P devices mounted on a vehicle can be used to accurately compute a heading angle (i.e. an alternative to compass-based heading estimation).
+The two GPS devices in this scenario are referred to as the *Moving Base* and *Rover*.
+
+### Supported Devices
+
+This is the subset of devices above that have a U-blox F9P:
+* [Freefly RTK GPS](../gps_compass/rtk_gps_freefly.md) (F9P)
+* [Holybro H-RTK F9P GNSS](../gps_compass/rtk_gps_holybro_h-rtk-f9p.md)
+* [SparkFun GPS-RTK2 Board - ZED-F9P](https://www.sparkfun.com/products/15136) (www.sparkfun.com)
+* [SIRIUS RTK GNSS ROVER (F9P)](https://store-drotek.com/911-1010-sirius-rtk-gnss-rover-f9p.html#/158-sensor-no_magnetometer) (store-drotek.com)
```

Comment:
As the UART2 needs to be exposed, the first 2 devices can't be used for heading.

There's also this: https://store.mrobotics.io/product-p/mr-m10020-a.htm that can be used.

Reply:
Thanks @bkueng . I added this one to the table, and to the top table. I also made comment that it isn't just F9P, but ability to access the UART2 which is relevant.